fhir-server-test/src/test/java/org/linuxforhealth/fhir/server/test/ServerSpecTest.java

```diff
@@ -60,6 +66,16 @@ public class ServerSpecTest extends FHIRServerTestBase {
     private Patient savedPatient;
     @SuppressWarnings("unused")
     private Observation savedObservation;
+    
+    private boolean updateCreateEnabled;
+    
+    @BeforeClass
+    public void setUp() throws Exception {
+        Properties testProperties = TestUtil.readTestProperties("test.properties");
+        setUp(testProperties);
+        updateCreateEnabled = isUpdateCreateSupported();
+        System.out.println("Update/Create enabled?: " + updateCreateEnabled);
```

Comment:
i assume this was mostly to help debug...would be good to either remove or make it conditional

Reply:
This has been fixed. Removed System.out.println statement.